cnf-tests/docgen/cmd/generate.go

```diff
@@ -124,10 +124,11 @@ func descriptionsToList(descriptions map[string]string, matcher func(string) boo
 		if !matcher(n) {
 			continue
 		}
+		n = sanitizeName(n)
```

Comment:
Can you use a different name here?

Something like `sanitizedName` := sanitazeName(n)
Reusing the variable may cause a bit of confusion